app/containers/SignInForm/reducer.js

```diff
@@ -0,0 +1,21 @@
+import { fromJS } from 'immutable';
+
+import { SIGN_IN_SUCCESS, SIGN_IN_FAILED } from './constants';
+
+const initialState = fromJS({
+  token: '',
```

Comment:
I remember we talked about storing the `token` at a higher level in order to be available to the entire app.